.github/ISSUE_TEMPLATE/release.yaml

```
@@ -0,0 +1,78 @@
+name:  Release
+description: Release/Pre-release of a version.
+title: "Release Version(e.g: 2.4.0)"
+labels: [release]
```

Let's be consistent with other lists' formatting

```suggestion
labels:
- release
```

---

This was the exemplar format on the page of issue forms in Github page: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository 
So I would keep it in this format.